release notes/v0.8.1.md

```diff
@@ -4,5 +4,17 @@ This is a patch release that fixes a few bugs, including one [regression bug](ht
 ## Bugs fixed
 
 - Fix mapping of Page's Keyboard, Mouse, and Touchscreen properties. [#751](https://github.com/grafana/xk6-browser/pull/751)
+- Fix usage of stale virtual user context in mapping layer. [#754](https://github.com/grafana/xk6-browser/pull/754).
```

Comment:
Not sure if anyone was impacted by this, so maybe best to place it in internal?

Reply:
Would we keep the wording in that case? E.g.: `Fix...`?

Reply:
Done in [7716840](https://github.com/grafana/xk6-browser/pull/775/commits/77168406e1124f202ccbcd05e1210b22e74a51a8).